test/test_conversion.py

```diff
@@ -62,10 +63,23 @@ def test_3to2_conversion(self):
         assert doc2.componentDefinitions[0].sequences[0] == 'https://synbiohub.org/public/igem/BBa_J23101_sequence'
         assert doc2.sequences[0].encoding == 'http://www.chem.qmul.ac.uk/iubmb/misc/naseq.html'
         assert doc2.sequences[0].elements == 'tttacagctagctcagtcctaggtattatgctagc'
+        # ids of location block containing orientation before conversion
+        location_ids_sbol3 = []
+        def change_if_location(o):
```

Comment:
This isn't actually changing locations: its name should describe what it actually does.